target/ppc/int_helper.c

```diff
@@ -1667,6 +1667,28 @@ void helper_xxinsertw(CPUPPCState *env, ppc_vsr_t *xt,
     *xt = t;
 }
 
+#define XXBLEND(name, sz) \
+void glue(helper_xxblendv, name)(ppc_avr_t *t, ppc_avr_t *a, ppc_avr_t *b,  \
+                                 ppc_avr_t *c, uint32_t desc)               \
```

Comment:
Since we don't use the last argument anywhere, maybe rename it to ``unused``?

I've noticed other parts of the code either:
- call the argument ``unused``
- use ``ATTRIBUTE_UNUSED``
- use ``G_GNUC_UNUSED``

Not sure what's the preferred form.

Reply:
I think it's not required, other GVec helpers are using `uint32_t desc` without any attributes even when unused. Also, it seems that qemu isn't using `-Wunused-parameter` (I can't even `./configure --extra-cflags="-Wunused-parameter"`) so it shouldn't make difference.